tests/backup/backup_kubevirt_test.go

```diff
@@ -0,0 +1,383 @@
+package tests
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	"github.com/pborman/uuid"
+	api "github.com/portworx/px-backup-api/pkg/apis/v1"
+	"github.com/portworx/torpedo/drivers/backup"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
+	"sync"
+	"time"
+)
+
+// This testcase verifies backup and restore of Kubevirt VMs in different states like Running, Stopped, Restarting
+var _ = Describe("{KubevirtVMBackupRestoreWithDifferentStates}", func() {
+
+	var (
+		backupNames                []string
+		restoreNames               []string
+		scheduledAppContexts       []*scheduler.Context
+		sourceClusterUid           string
+		cloudCredName              string
+		cloudCredUID               string
+		backupLocationUID          string
+		backupLocationName         string
+		backupLocationMap          map[string]string
+		labelSelectors             map[string]string
+		providers                  []string
+		restoreNameToAppContextMap map[string]*scheduler.Context
+		namespaceMappingMixed      map[string]string
+		namespaceMappingRestart    map[string]string
+		namespaceMappingStopped    map[string]string
+		backupWithVMRestart        string
+		restoreWithVMRestart       string
+		namespaceWithStoppedVM     []string
+		backupWithVMMixed          string
+		restoreWithVMMixed         string
+		backupWithVMStopped        string
+		restoreWithVMStopped       string
+	)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("KubevirtVMBackupRestoreWithDifferentStates", "Verify backup and restore of Kubevirt VMs in different states",
+			nil, 93011)
```

Comment:
Please put all params in one line. It would be easy for automation tracker to get TestRailID.

Reply:
Done